.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+<!-- Thank you for your contribution! -->
+
+<!-- Please add a reviewer to the assignee section when you create a PR. If you don't have the access to it, we will shortly find a reviewer and assign them to your PR. -->
+
+## Why are these changes needed?
+
+<!-- Please give a short summary of the change and the problem this solves. -->
+
+## Related issue number
+
+<!-- For example: "Closes #1234" -->
+
+## Checks
+
+- [ ] I've made sure the tests are passing. 
+- Testing Strategy
+   - [ ] Unit tests
+   - [ ] Release tests
```

Comment:
We don't have a release test yet.  Let's rename to "Manual tests"

Reply:
Sure. I will change to that.